tutorial/index.html

```diff
@@ -0,0 +1,412 @@
+<html>
+<head>
+    <title>Akka HTTP to Persistent Actor to Akka Stream to Websocket</title>
+</head>
+<body>
+<div>
+    <h2>Getting started with this template</h2>
+
+    <p>
+        In this template you will learn some basics of
```

Comment:
"the basics" instead of "some basics"? Don't be too self-deprecating :). Basics are subjective anyway.